discovery/ec2/ec2.go

```diff
@@ -257,13 +257,20 @@ func (d *Discovery) refresh() (tg *targetgroup.Group, err error) {
 				if inst.VpcId != nil {
 					labels[ec2LabelVPCID] = model.LabelValue(*inst.VpcId)
 
+					// Deduplicate VPC Subnet IDs maintaining the order of the network interfaces returned by EC2
+					// The Subnet ID from the instance metadata (if available) will always be first on the list
+					// This can help with relabeling if the user is only interested on the main Subnet
```

Comment:
nit picking but all sentences need to end with a dot.

Reply:
Done.